### ISSUE TITLE
No main.py created in windows??

### ISSUE THREAD

**Issue by slormo** [kensanata/mastodon-backup#77]
After installing the repo as instructed, I ran the command to test it.

`C:\Users\****>python3 mastodon-archive archive kensanata@dice.camp`

This returned the error:

```
C:\Users\****\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\python.exe: can't find '__main__' module in 'C:\\Users\\****\\mastodon-archive'
```

It looks like a main.py file was not created. Any help?

**Comment by kensanata** [comment]
Sadly, I have no idea and never tried to install it on a Windows machine. Anybody else?

**Comment by IzzySoft** [comment]
Maybe we should include the one created by `contrib/debinstall.sh` as a file for such cases, so users can simply copy it over manually (would also make packaging easier and the SPEC file for that cleaner)? All windows I have are just for light and fresh air, and the prefixed "MasterShutter" ("MS for my Windows") is a Somfy remote control, so I cannot test there either.

**Comment by IzzySoft** [comment]
@slormo can you pick the `contrib/mastodon-archive.py` from the linked PR, place that into the `mastodon-archive/` directory and see if that works for you?

**Comment by slormo** [comment]
Nope, still doesn't work I'm afraid.

I tried the command, and got this error still.
```
C:\Users\****>python3 mastodon-archive archive --no-favourites --no-bookmarks kensanata@dice.camp
C:\Users\****\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\python.exe: can't find '__main__' module in 'C:\\Users\\****\\mastodon-archive'
```
Here's my directory, in case I've done anything wrong.
![Screenshot 2022-11-02 234211](https://user-images.githubusercontent.com/104801833/199622905-8224865b-1f9f-4604-953a-bc8ef4625346.png)

**Comment by IzzySoft** [comment]
Yes, `mastodon-archive.py` must be one level up: next to the `mastodon-archive` directory, not inside.

**Comment by slormo** [comment]
I've moved it to the correct location, but it still throws up the exact same error.

**Comment by IzzySoft** [comment]
Did you call it with

    python3 mastodon-archive.py archive --no-favourites --no-bookmarks <account>

i.e. include the `.py` so Python knows to use the **file** and not the **directory** directly?
